src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -86,6 +91,10 @@ class NUnit extends ConventionTask {
         if (getIsV3(version)) {
             this.metaClass.mixin NUnit3Mixins
         }
+        else
+        {
```

Comment:
should follow the same code convention, having curly bracket on the previous line

Reply:
done